### PR TITLE
LPD-28035 - Create an Endpoint to retrieve all the accounts in an account group

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-28035

### CODE REVIEW

**Comment by matyaswollner** [comment]
ci:test:sf

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java [hunk 1]
```diff
@@ -148,6 +164,46 @@ public void testDeleteOrganizationAccountsByExternalReferenceCode()
...
+
+	@Override
+	protected String
+	testGetAccountGroupByExternalReferenceCodeAccountsPage_getAccountGroupExternalReferenceCode()
+		throws Exception {
+
+		return _accountGroup.getExternalReferenceCode();
+	}
+
+	protected Account
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java:194]: missing override

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java:1]: missing SF on this file

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java:1]: missing SF on this file

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java [hunk 2]
```diff
@@ -149,6 +154,58 @@ public Account getAccountByExternalReferenceCode(
...
+				contextCompany.getCompanyId()),
+			sorts,
+			document -> _toAccount(
+				Collections.emptyMap(),
+				GetterUtil.getLong(document.get(Field.ENTRY_CLASS_PK))));
+	}
+
+	@Override
+	public Page<Account> getAccountGroupByExternalReferenceCodeAccountsPage(
+			String externalReferenceCode, String search, Filter filter,
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java:191]: externalReferenceCode -> accountGroupExternalReferenceCode (like the parameter in the yaml - and in the base class)

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:stable

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**